addons/website/models/digest.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class Digest(models.Model):
+    _inherit = 'digest.digest'
+
+    kpi_website_visitor_cnt = fields.Boolean('Visitors')
+    kpi_website_visitor_cnt_value = fields.Integer(compute='_compute_kpi_website_visitors_cnt_value')
+    kpi_tracked_page_view_cnt = fields.Boolean('Tracked page views')  # Non uniq
+    kpi_tracked_page_view_cnt_value = fields.Integer(compute='_compute_kpi_tracked_page_view_cnt_value')
+
+    def _compute_visitor_cnt(self, ctx):
+        website_ids, start, end = ctx
+        if len(website_ids) == 0:
+            return 0
+        # The query takes into account that some website_visitor are the same visitor through parent_id
+        self._cr.execute("""
+            SELECT count(distinct COALESCE(v.parent_id, v.id))
```

Comment:
Is that necessary ? I would have two main thoughts
1/ we aggregate statistics on the main partner
2/ website team is working to remove parent_id anyway (quite sure it will disappear)

Reply:
Since that time, the cleaning of visitor has been merged. I think we spoke about it. Globally there is no parent_id field anymore, meaning we don't have 'archived' or 'duplicate children' anymore. This should simplify the computation.